app/models/spree/review.rb

```diff
@@ -27,6 +27,23 @@ class Spree::Review < ApplicationRecord
   scope :not_approved, -> { where(approved: false) }
   scope :default_approval_filter, -> { Spree::Reviews::Config[:include_unapproved_reviews] ? all : approved }
 
+  def self.ransackable_attributes(*)
```

Comment:
What about using [Spree::RansackableAttributes](https://github.com/solidusio/solidus/blob/main/core/app/models/concerns/spree/ransackable_attributes.rb) instead? I'm thinking that may be the "conventional" way to change `ransackable_attributes` and similar in our ecosystem.

As far as I remember, this would become something like:
```ruby
extend Spree::RansackableAttributes
self.allowed_ransackable_attributes = %w[approved name review title]
```